src/App/Fossa/Analyze.hs

```diff
@@ -373,13 +384,13 @@ analyze cfg = Diag.context "fossa-analyze" $ do
           (Just firstParty, Just lernie) -> Just $ firstParty <> lernie
           (Nothing, Just lernie) -> Just lernie
           (Just firstParty, Nothing) -> Just firstParty
-  let result = buildResult includeAll additionalSourceUnits filteredProjects licenseSourceUnits
+  let result = buildResult includeAll additionalSourceUnits filteredProjects' licenseSourceUnits
   let keywordSearchResultsFound = (maybe False (not . null . lernieResultsKeywordSearches) lernieResults)
 
   -- If we find nothing but keyword search, we exit with an error, but explain that the error may be ignorable.
   -- We do not want to succeed, because nothing gets uploaded to the API for keyword searches, so `fossa test` will fail.
   -- So the solution is to still fail, but give a hopefully useful explanation that the error can be ignored if all you were expecting is keyword search results.
-  case (keywordSearchResultsFound, checkForEmptyUpload includeAll projectResults filteredProjects additionalSourceUnits licenseSourceUnits) of
+  case (keywordSearchResultsFound, checkForEmptyUpload includeAll filteredProjects' filteredProjects' additionalSourceUnits licenseSourceUnits) of
```

Comment:
I don't understand why we're sending `filteredProjects'` twice into `checkForEmptyUpload`. It feels like a bug, maybe not introduced in this PR, but could we fix it here if it's easy?

If that's coming in the future PR, that's all good. Just wanted to double-check